FreeRTOS/FreeRTOS-Plus-TCP
# [IPv6] Unify IPv6 UT file format

[IPv6] Unify IPv6 UT file format

Description
-----------

To unify the files format for IPv6, including:
* FreeRTOS_BitConfig
* FreeRTOS_DHCPv6
* FreeRTOS_IPv6
* FreeRTOS_IPv6_Utils
* FreeRTOS_ND
* FreeRTOS_RA
* FreeRTOS_Routing
* FreeRTOS_Sockets_IPv6
* FreeRTOS_UDP_IPv6
* FreeRTOS_TCP_Transmission_IPV6

Test Steps
-----------

Run all unit test cases after changing.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
